packages/eslint-plugin/src/rules/comma-spacing.ts

```diff
@@ -110,6 +110,7 @@ export default createRule<Options, MessageIds>({
       if (
         prevToken &&
         isTokenOnSameLine(prevToken, commaToken) &&
+        // eslint-disable-next-line deprecation/deprecation -- TODO - switch once our min ESLint version is 6.7.0
```

Comment:
Sorry if I'm forgetting something obvious, what was the line in the sand on ESLint versions here? Is it documented somewhere else?

Reply:
Oh I see I found it on eslint.org, `isSpaceBetweenTokens() - replaced by isSpaceBetween()`

Reply:
yeah it's marked as deprecated in our types.
in the next major we'll probably do a massive bump to a higher ESLint version (maybe 6.7?) to enable us to switch to these new APIs and clean house as well.